src/app/personside/visittkort/header/EtiketterContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Egenansatt } from '../../../../models/egenansatt';
+import Innholdslaster from '../../../../components/Innholdslaster';
+import { AppState, Reducer } from '../../../../redux/reducer';
+import { Person } from '../../../../models/person';
+import Etiketter from './Etiketter';
+
+interface Props {
+    egenAnsattReducer: Reducer<Egenansatt>;
+    personReducer: Reducer<Person>;
+}
+
+const onError = (
+    <em>Problemer med å hente eventuell egenAnsatt-markering</em>
```

Comment:
Litt pirk: endre denne til problemer med å vise etikketter? Siden den egentlig er generisk for begge casene.

Reply:
se der, så ikke denne kommentaren isted :) Enig, I fix!